database/migrations/0053-change_password_encryption.ts

```diff
@@ -0,0 +1,22 @@
+/* MIGRATION TO ADD ENCRYPTION TYPE TO PASSWORDS
+ *
+ * This migration adds and renames columns in the table `users`
+ */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn('ALTER TABLE users RENAME COLUMN created TO created_at;')
+  await queryFn('ALTER TABLE users RENAME COLUMN deletedAt TO deleted_at;')
+  // alter table emp rename column emp_name to name
+  await queryFn(
+    'ALTER TABLE users ADD COLUMN password_encryption_type int(10) NOT NULL DEFAULT 0 AFTER password;',
+  )
```

Comment:
I am missing something like
`UPDATE users SET password_encryption_type = 1 WHERE id IN (SELECT user_id FROM user_contacts WHERE email_checked = 1);

Reply:
May be it is even better to join users and user contacts on email_id

Reply:
@ulfgebhardt We have to check this migration on stage 2 carefully

Reply:
`UPDATE u SET password_encryption_type = 1 FROM users u LEFT JOIN user_contacts uc ON u.id = uc.user_id WHERE uc.email_checked = 1;`
(untested)